app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+
+    has_many :events, foreign_key: "creator_id", class_name: "Event"
+    has_many :attended_events, foreign_key: "attendee_id"
+    has_many :event_attendance, through: :attended_events, source: :attended_event
+
+
+    def upcoming_events
```

Comment:
Let's be more explicit in naming this method. Since a user can have two types of event (as `creator` and as `attendee`) then we should probably name this method `upcoming_events_to_attend`